.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         ruby: [2.7]
-        idna_mode: [native, pure]
+        idna_mode: [native2, native, pure]
```

Comment:
Don't we also need to install Libidn2 in CI?

Reply:
Apparently not, looks like [`libidn2`](https://packages.ubuntu.com/focal/libidn2-0) is probably a dependency to [many packages installed on the runner images](https://github.com/actions/runner-images/blob/833ffa3aab68f80abf0b0e987e212f43da58f925/images/linux/Ubuntu2004-Readme.md#installed-apt-packages), e.g. [libcurl4](https://packages.ubuntu.com/bionic/libcurl4) or [libc6](https://packages.ubuntu.com/focal/libc6)